board/piksiv3/rootfs-overlay/etc/init.d/S75dev_boot

```diff
@@ -23,8 +23,8 @@ copy_from_sd() {
 copy_from_net() {
   local server_ip=$1
   local filename=$2
-  local serial_number=`cat /factory/serial_number`
-  local net_file="/PK$serial_number/$filename"
+  local uuid=`cat /factory/uuid`
+  local net_file="/PK$uuid/$filename"
```

Comment:
I do not care. Whatever you guys think.

FYI the UUID is stored (and used above) without any dashes or anything.